mkdocs.yml

```diff
@@ -14,19 +14,20 @@ nav:
     - Linux Headers: install/headers.md
   - Integrations: integrations.md
   - Configuration Options: config.md
-  - Rules Authoring: rules-authoring.md
-  - Rules Index: rules-index.md
+  - Authoring Rules: rules-authoring.md
+  - Available Rules: rules-index.md
   - FAQ: faq.md
+  - Architecture: architecture.md
   - Tracing:
     - Getting Started: tracee-ebpf/index.md
     - Output Options: tracee-ebpf/output.md
-    - Trace Options: tracee-ebpf/trace.md
+    - Trace Options: tracee-ebpf/trace-options.md
     - Capturing Artifacts: tracee-ebpf/capture.md
 
 theme:
     name: material
     language: 'en'
-    logo: images/tracee.png
+    logo: images/tracee_logo_only.png
```

Comment:
Seems like we have two asset sources now, one under `docs/images/` and one under `images/` could we reconcile them somehow?

Reply:
This PR removes the `images/` directory for the base of the repo completely. This particular line is referencing the one in `docs/images/`